drift/info_parser.py

```diff
@@ -0,0 +1,43 @@
+DRIFT_FACTS = {'bios_uuid', 'display_name'}
+DRIFT_HOST_ID = {'id'}
+
+
+def select_applicable_info(info_keys, hosts):
+    """
+    this pivots a list of hosts with dicts of info + values into a list of info
+    with dicts of hosts + values
+    """
+    # create dicts of host + info
+    ids_and_info = [{key: host[key] for key in (info_keys | DRIFT_HOST_ID & host.keys())}
+                    for host in hosts]
+
+    # TODO: we are assuming all info keys exist for each host. This is not a good assumption
+    available_info_names = ids_and_info[0].keys()
+
+    info_comparisons = [create_comparison(ids_and_info, info_name)
+                        for info_name in available_info_names
+                        if info_name in info_keys]
+    return info_comparisons
+
+
+def create_comparison(hosts_infos, info_name):
```

Comment:
The plural seems unneeded on `infos`. What is `info_name` in this respect?

Reply:
it is the key for the item coming back from inventory service. I originally called it `fact_name` but not all pieces of information coming back are facts.

I'll add a docblock to these methods, they are kinda tricky to understand

Reply:
docblocks added